lab-michael/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "lab-08-single-resource-api",
+  "version": "1.0.0",
+  "description": "pulling a resource from api",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha",
+    "start": "node server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/silvershotm1/lab-08-single-resource-api.git"
+  },
+  "author": "Michael Bishop",
+  "license": "MIT",
+  "devDependencies": {
+    "bluebird": "^3.4.6",
```

Comment:
bluebird is not a dev dep given that it's a requirement of your app functionality.